compiler/rustc_mir_transform/src/simplify.rs

```diff
@@ -36,13 +36,36 @@ use rustc_middle::mir::*;
 use rustc_middle::ty::TyCtxt;
 use smallvec::SmallVec;
 
-pub struct SimplifyCfg {
-    label: String,
+static SIMPLIFIED_CFG_CONDITIONS: [&'static str; 8] = [
+    "SimplifyCfg-initial",
+    "SimplifyCfg-promote-consts",
+    "SimplifyCfg-remove-false-edges",
+    "SimplifyCfg-early-opt",
+    "SimplifyCfg-elaborate-drops",
+    "SimplifyCfg-final",
+    "SimplifyCfg-make_shim",
+    "SimplifyCfg-after-uninhabited-enum-branching",
+];
+
+pub struct SimplifyCfg<'a> {
+    label: &'a str,
```

Comment:
You can just store a `&'static str` and remove the lifetimes

Reply:
Oh neat, thanks. 
Would those be guaranteed to only be instantiated once?

Reply:
`&'static str` are typically (except for [exceptional cases](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=65fc6e5faf6b6eb8d2fe452aa16b571d)) statically allocated, so yeah.